js/Navigation.js

```diff
@@ -1,40 +1,17 @@
 export default function Nav() {
-  const homeButton = document.querySelector('[data-js="home-button"]');
-  const homeContent = document.querySelector('[data-js="home-content"]');
-  const createButton = document.querySelector('[data-js="create-button"]');
-  const createContent = document.querySelector('[data-js="create-content"]');
-  const bookmarkButton = document.querySelector('[data-js="bookmark-button"]');
-  const bookmarkContent = document.querySelector(
-    '[data-js="bookmark-content"]'
-  );
-  const profileButton = document.querySelector('[data-js="profile-button"]');
-  const profileContent = document.querySelector('[data-js="profile-content"]');
+  const allNavIcons = document.querySelectorAll('[data-nav]');
+  const allPages = document.querySelectorAll('[data-page]');
 
-  homeButton.addEventListener('click', () => {
-    homeContent.classList.remove('hide');
-    createContent.classList.add('hide');
-    bookmarkContent.classList.add('hide');
-    profileContent.classList.add('hide');
-  });
-
-  createButton.addEventListener('click', () => {
-    createContent.classList.remove('hide');
-    homeContent.classList.add('hide');
-    bookmarkContent.classList.add('hide');
-    profileContent.classList.add('hide');
-  });
-
-  bookmarkButton.addEventListener('click', () => {
-    bookmarkContent.classList.remove('hide');
-    homeContent.classList.add('hide');
-    createContent.classList.add('hide');
-    profileContent.classList.add('hide');
-  });
-
-  profileButton.addEventListener('click', () => {
-    profileContent.classList.remove('hide');
-    homeContent.classList.add('hide');
-    createContent.classList.add('hide');
-    bookmarkContent.classList.add('hide');
+  allNavIcons.forEach((clickedIcon) => {
+    clickedIcon.addEventListener('click', (event) => {
+      allPages.forEach((otherPages) => {
```

Comment:
Why did you name the parameter otherPages? It might seem a bit confusing. I would just name it 'pages' as the forEach function will run for all pages which are listed inside allPages.

Reply:
haha thank you, you are right 😅